src/javascripts/modules/pills.js

```diff
@@ -0,0 +1,40 @@
+export default class Pills {
+  constructor(el) {
+    this.el = el
+    this.setupListeners()
+  }
+
+  setupListeners() {
+    window.addEventListener('load', () => this.handleLoad())
+  }
+
+  handleLoad() {
+    clearTimeout(this.timeout)
+    this.timeout = setTimeout(() => this.animatePills(), 500)
+    this.releaseSqueezeTimeout = setTimeout(() => this.releasePillSqueeze(), 4500)
+  }
+
+  animatePills() {
+    if (this.el.classList.contains('yellow-pill')) {
+      this.el.classList.add('yellow-pill-roll')
+    } else if (this.el.classList.contains('more-pills')) {
+      this.el.classList.add('more-pills-roll')
+    } else if (this.el.classList.contains('purple-pills')) {
+      this.el.classList.add('purple-pills-roll')
+    } else if(this.el.classList.contains('title-tablet')) {
+      this.el.classList.add('title-tablet-roll')
+    } else if(this.el.classList.contains('title-pill-red')) {
+      this.el.classList.add('title-pill-red-bounce')
```

Comment:
One option for clearing up repetition here is to use another data attribute that specifies what type of effect you want, roll or bounce. So it could look something like this:
```
<img ... data-module="yellow-pill" data-effect="roll">
```
and:
```
const effectClassname = `${this.el.className}-${this.el.dataset.effect}`
this.el.classList.add(effectClassname)
```

Reply:
Oops, my first code snippet there is wrong; will fix.

Reply:
Fixed.